tests/http/batching.yml

```diff
@@ -72,3 +72,19 @@ assert:
               - column: 1
                 line: 1
             message: " --> 1:1\n  |\n1 | FOO\n  | ^---\n  |\n  = expected executable_definition"
+  # Sending a bad request
+  - request:
+      method: GET
+      url: http://localhost:8080/api/posts\?\$\=title,user.name
```

Comment:
does yaml requires escaping all that symbols?

even if so, I think it could be fixed by wrapping the whole string in quotes